mkdocs/cli.py

```diff
@@ -120,13 +122,14 @@ def json_command(clean, config_file, strict, site_dir):
 @cli.command(name="gh-deploy")
 @click.option('--clean', is_flag=True, help=clean_help)
 @click.option('--config-file', type=click.File('rb'), help=config_file_help)
-def gh_deploy_command(config_file, clean):
+@click.option('--commit-message', is_flag=False, help=commit_message_help)
```

Comment:
I realize this is bikeshedding, but why not follow Git's pattern (like most Git add-ons do) and use `--message`. I would think all Git users will expect either that or `-m`. Of course, Click can support both, which is my personal preference (yes I realize mkdocs does not in most cases - to my annoyance). The point here is that `--message` would be the least surprising and the easiest to remember for users. The help text identifies it specifically as a *commit* message.